osquery/tables/system/windows/background_activities_moderator.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/core/core.h>
+#include <osquery/core/tables.h>
+#include <osquery/logger/logger.h>
+#include <osquery/tables/system/windows/registry.h>
+#include <osquery/utils/conversions/windows/windows_time.h>
+
+#include <string>
+
+namespace osquery {
+namespace tables {
+
+constexpr auto kBamRegPath =
+    "HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\bam\\%%\\%%";
```

Comment:
The implication here, is that this is a registry entry, and that this table primarily reformats the registry for nicer display?

Is that correct? If so, I'm ambivalent... One of the things I've found, is that it can lead to a large number of specific tables, where I favor a smaller number of compostable tables. Another issue is that it can create maintenance burdens. What happens if the format changes?

Reply:
I'm withdrawing this objection. We talked about in office hours, and there is not consensus. Table seems useful!